client/azure/azure.go

```diff
@@ -0,0 +1,132 @@
+package azure
```

Comment:
I'm fine with flattening the structure. I think client.NewAzureClientSet is a good name for it.